### PR TITLE
Core/add chicago project

### LINKED ISSUE #75: New Project "Chicago"
We had to add the new project to the plataform, all the assets are in your blick mail

### LINKED ISSUE #78: Remove blog from navbar

### PR DESCRIPTION
**Pull request by AlanSanchezP**
This pull request aims to resolve #75 and to resolve #78.

### CODE REVIEW

File: vre/templates/footer.html [hunk 1]
```diff
@@ -62,7 +62,7 @@
     </div>
     <div class="footer-col text-center">
       <img src="{% static "img/icons/vre_full_white_logo.png" %}" alt="VRE Logo" class="footer-logo">
-      <span class="footer-logo-legend">&copy; VRE 2016</span>
+      <span class="footer-logo-legend">&copy; VRE 2017</span>
```
Comment (jresendiz27) [inline vre/templates/footer.html]: Change this to use a date object maybe in python or JS

File: vre/templates/develops/develop_detail_5.html [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    de la colonia Nápoles, désde 62 m2.
+  </div>
+  <div class="gray-container development-download-brochure">
+    <div>
+      <span>Si quieres más información, descarga aquí nuestro brochure de ventas</span>
+      <form id="download-brochure-form">
+        {% csrf_token %}
+        <input type="email" id="download-brochure-email" placeholder="Correo electrónico" name="email">
+        <input type="hidden" name="source" value="chicago">
+	      <span id="download-brochure-span" style="display:block;"></span>
```
Comment (jresendiz27) [inline vre/templates/develops/develop_detail_5.html]: I'm not a CSS expert, but inline css it's a bad practice, @Enrique199413 right?

**Review by jresendiz27 — CHANGES_REQUESTED** [review]

File: vre/templates/develops/develop_detail_5.html (hunk 2 above)
Comment (Enrique199413) [reply vre/templates/develops/develop_detail_5.html]: Yes is a bad practice.

http://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css

File: vre/templates/footer.html (hunk 1 above)
Comment (Enrique199413) [reply vre/templates/footer.html]: For the next year changed automatically.

Brilliant!

**Review by Enrique199413 — COMMENTED** [review]
Please resolve all comments

**Pull request merged**